### ISSUE TITLE
Create capstone project board

### ISSUE THREAD

**Issue by King-Benx** [King-Benx/your-movies-capstone-project#10]
**Delivers:**
Project board

**Link:**
[https://github.com/users/King-Benx/projects/3](https://github.com/users/King-Benx/projects/3)

This is the project kanban for the meals capstone project we will be doing

**Members:**

1. [Dilsher](https://github.com/DilsherB)
2. [Ben](https://github.com/King-Benx)

**Link to project:**
[https://github.com/King-Benx/meal-capstone-project](https://github.com/King-Benx/meal-capstone-project)

**Comment by didierganthier** [comment]
Hi @King-Benx and @DilsherB ,

While you put in a lot of hard work in this project, unfortunately, I won't be able to proceed with the review of your code.

<img src="https://media.giphy.com/media/HOTfexLzbar5u/giphy.gif" />

## Invalid Code Review Request 🚫

- You have submitted a Kanban board that doesn't give information about the number of people in their group.

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Comment by didierganthier** [comment]
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.

<img src="https://media.giphy.com/media/3oEduMcvxTNPlGuhH2/giphy.gif" />

## Highlights
- Kanban setup ✔️ 
- Issues assigned ✔️ 
- Project created ✔️ 

## Required Changes 🛠
- [ ] Well done on the Kanban board 💯 but since this is a team with 2 people, and all cards with Student C are to be removed, the number of cards on board should be equal to 15. Please refer to the [template](https://github.com/microverseinc/curriculum-javascript/projects/1)
- [ ] Good job assigning the cards 👍🏽 but cards should be assigned to team members in a fair way, @King-Benx's assigned tasks equal to 2 points while @DilsherB's assigned tasks equal to 3 points (there's one card assigned to him that doesn't have any points). 🤔 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Comment by m05e5** [comment]
**Hi Team 
You did great. Your project is complete!  There is nothing else to say other than... it's time to start coding:shipit: 
Congratulations! 🎉 
![CleverScornfulBuffalo-size_restricted](https://user-images.githubusercontent.com/37901469/208869296-5ca9f6cc-a1b5-48fd-bd59-78fd83b35471.gif)

## Highlights

✔️ Clean kanban board

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
